pkgs/by-name/gi/git-unroll/package.nix

```
@@ -0,0 +1,81 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitea,
+  makeWrapper,
+  bash,
+
+  git,
+  nix,
+  rWrapper,
+  rPackages,
+
+  unstableGitUpdater,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "git-unroll";
+  version = "0-unstable-2024-11-04";
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "gm6k";
+    repo = "git-unroll";
+    rev = "9243bb8a6a9f6875e21a5c64320b66f7fdaf9b3f";
+    hash = "sha256-1MjbB1EVgmU0HlUibrKOkjmxQ8wseocSJENiAqyHcjU=";
+  };
+
+  postPatch = ''
+    substituteInPlace unroll \
+      --replace-fail "#! /usr/bin/env nix-shell" "#!/usr/bin/env bash" \
+      --replace-fail \
+        "#! nix-shell -i bash -p git nix rWrapper rPackages.jsonlite rPackages.processx rPackages.dplyr rPackages.plyr rPackages.stringr -I nixpkgs=." \
+        "" \
+      --replace-fail '"$PWD/pkgs/build-support/fetchgit/nix-prefetch-git",' '"nix-prefetch-git"'
+  '';
+
+  nativeBuildInputs = [
+    makeWrapper
+  ];
+
+  buildInputs = [
+    bash
+  ];
+
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm755 unroll $out/bin/unroll
+    wrapProgram $out/bin/unroll \
+      --prefix PATH : ${
+        lib.makeBinPath [
+          git
+          nix
```

```suggestion
          nix-prefetch-git
```
It only uses the prefetcher, which is not in nix I think